packages/x-components/src/x-modules/facets/store/actions/set-filters-from-url.action.ts

```diff
@@ -0,0 +1,29 @@
+import { createRawFilters, Dictionary } from '../../../../utils';
+import { UrlParamValue } from '../../../url';
+import { FacetsXStoreModule } from '../types';
+
+/**
+ * Default implementation for the {@link FacetsActions.setUrlParamsFromTheUrl}.
+ *
+ * @param context - The {@link https://vuex.vuejs.org/guide/actions.html | context} of the actions,
+ * provided by Vuex.
+ * @param urlParams - The url params from the url.
+ *
+ * @public
+ */
+export const setUrlParamsFromTheUrl: FacetsXStoreModule['actions']['setUrlParamsFromTheUrl'] = (
```

Comment:
Why is this file called `set-filters-from-url.action`, but the exported method `setUrlParamsFromTheUrl`?. Both names should match the action name

Reply:
Fixed.